jerry-debugger/jerry-client-ws.py

```diff
@@ -348,22 +348,30 @@ def do_delete(self, args):
     def do_backtrace(self, args):
         """ Get backtrace data from debugger """
         max_depth = 0
+        min_depth = 0
 
         if args:
+            args = args.split(" ")
             try:
-                max_depth = int(args)
-                if max_depth <= 0:
+                min_depth = int(args[0])
+                max_depth = int(args[1])
```

Comment:
What happens if there is` args` length is 1 and there is no second element?
Both `bt max_value` and `bt min_value max_value` commands should work.

Reply:
I see now, this is intentional and the `bt` command has two mandatory arguments. IMHO this should be changed. One argument should mean a `max_value` and two arguments should mean `min_value max_value` pair.

Reply:
I don't mind having two mandatory arguments.

Reply:
Sorry I misunderstood you. Yes, in the python debugger, only one argument must be mandatory. In the message, both values must be sent.